### PR TITLE
docs(definitions): minor formatting to definitions' types

### PR DESCRIPTION
**Pull request by hankucz**
Align definitions documentation...

### CODE REVIEW

File: src/definitions/hacker.ts [hunk 1]
```diff
@@ -8,26 +8,31 @@ export type HackerDefinitions = LocaleEntry<{
...
-   * Some computer related nouns (e.g. `protocol`, `sensor`)
+   * Some computer related nouns (e.g. `protocol`, `sensor`).
    */
   noun: string[];
+
   /**
    * Some phrases that will be injected with random hacker words.
    * May use any of the HackerDefinition keys wrapped in double braces.
-   * (e.g. `I'm {{ingverb}} {{adjective}} {{noun}}` )
+   * (e.g. `I'm {{ingverb}} {{adjective}} {{noun}}` ).
```
Comment (ST-DDT) [inline src/definitions/hacker.ts:30]: If you add a dot after the parenthesis, you should remove the dot before them.

**Review by ST-DDT — COMMENTED** [review]
Nice!

File: src/definitions/hacker.ts (hunk 1 above)
Comment (hankucz) [reply src/definitions/hacker.ts]: Done! Thanks...

**Review by ST-DDT — APPROVED** [review]

**Review by pkuczynski — APPROVED** [review]

**Review by xDivisionByZerox — APPROVED** [review]
This is big for consistency!

**Pull request merged**